projects/kaltura-ng/kaltura-ui/src/lib/dynamic-form/dynamic-form.service.ts

```diff
@@ -40,7 +40,7 @@ export class DynamicFormService {
                             result = this.toFormGroup(formControl.children, {formValue: config.formValue});
                         }
                     } else {
-                        result[formControl.key] = new FormControl(formControl.defaultValue || null, formControl.validators);
+                        result[formControl.key] = new FormControl(formControl.defaultValue || null, {validators: formControl.validators, updateOn: formControl.validateOn ? formControl.validateOn : 'change'} );
```

Comment:
@tanyaLibatter - second parameter should be a ValidatorFn or an array of ValidatorFn. Not sure how passing an object works. Let's discuss.